components/adminTable.js

```diff
@@ -99,6 +123,94 @@ export default function AdminTable(props) {
             );
           })}
         </tbody>
+        <tfoot>
+          <tr>
+            <td colSpan='4' style={{ textAlign: 'right' }}>
+              <div
+                style={{
+                  marginTop: '10px',
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'flex-end',
+                  color: '#757575'
+                }}
+              >
+                <label>
+                  Rows per page:
+                  <select
+                    value={entriesPerPage}
+                    onChange={e => setEntriesPerPage(Number(e.target.value))}
+                  >
+                    <option value={10}>10</option>
+                    <option value={20}>20</option>
+                    <option value={30}>30</option>
+                    <option value={40}>40</option>
+                    <option value={50}>50</option>
+                  </select>
+                </label>
+                <span
+                  style={{
+                    marginLeft: '20px',
+                    marginRight: '10px',
+                    color: '#757575'
+                  }}
+                >
+                  {startEntry}-{endEntry} of {totalEntries}
+                </span>
+                <button
+                  onClick={() => setPageIndex(0)}
+                  disabled={!canPreviousPage}
+                  style={{
+                    marginLeft: '10px',
+                    marginRight: '10px',
+                    color: '#d1d1d1',
+                    fontSize: '1.5em'
+                  }}
+                >
+                  |<b>&lt;</b>
```

Comment:
I see a lot of repeated styling code in the buttons, would we be able to cut down on this?